source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+
+Begin by creating a directory to conduct the work in:
+
+.. command-block::
+   :no-exec:
+
+   mkdir qiime2-metadata-tutorial
+   cd qiime2-metadata-tutorial
+
+Metadata from a mapping file
+----------------------------
+
+Metadata is typically defined in a mapping file, similar to the QIIME 1 mapping file. The QIIME 2 development team hasn't adopted a standard set of criteria for the metadata mapping file, but at present the following minimum requirements are enforced:
+
+- The file must be a tab-separated text file
+- The first line of the file is the column labels, tab-separated
+- The header values in the first line must be unique (i.e. no duplicate values)
+- There is at least one line of data following the header line
+- The first column is the "identifier" column (either Sample ID or Feature ID, depending on the axis)
+- All of the values in the first column must be unique (i.e. no duplicate values)
+
+Metadata mapping files can be validated using Keemei_. Generally speaking, metadata mapping files `that work in QIIME 1`_ should work in QIIME 2.
+
+To get started with understanding metadata mapping files, first download a sample mapping file:
+
+.. TODO: Update this link
+.. download::
+   :url: https://data.qiime2.org/2017.5/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: sample-metadata.tsv
+
+This file can be opened and edited in a text editor, Google Sheets, or Keemei_. There is also a ``metadata tabulate`` visualization in QIIME 2 that allows for interactive sorting and filtering of metadata:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv
+     --o-visualization tabulated-sample-md.qzv
+
+.. question::
+   Based on the table in ``tabulated-sample-md.qzv``, how many samples are associated with Subject 1? How many samples are associated with the ``gut`` body site? Hint: use the search box and/or the column sorting options to assist with this query.
+
+Metadata from a QIIME 2 artifact
+--------------------------------
+
+As eluded to above, QIIME 2 also supports the notion of viewing artifacts as metadata (where it makes sense). An example of this is alpha-diversity output artifacts.
+
+To get started with understanding metadata mapping files, first download a sample mapping file:
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/core-metrics-results/faith_pd_vector.qza
+   :saveas: faith_pd_vector.qza
+
+To view this artifact as metadata, simply pass it in to any method or visualizer that expects to see metadata (e.g. ``metadata tabulate``):
```

Comment:
I think you should also mention generating an Emperor plot with this here, and then coming back to it later (which you're doing now). That's a pretty cool use case that will illustrate the power of this. How about: *(e.g. ``metadata tabulate`` or ``emperor plot``)*